requirement.txt

```diff
@@ -0,0 +1,3 @@
+django=4.0.1
+django_crispy_forms
+django-filter
```

Comment:
to avoid compatibilities issues in the future always define the version number after the name, like django=x.x.x